ludwig/models/ecd.py

```diff
@@ -42,7 +42,7 @@ def __init__(
         except KeyError as e:
             raise KeyError(
                 f"An input feature has a name that conflicts with a class attribute of torch's ModuleDict: {e}"
-            )
+            ) from e
```

Comment:
Should we get rid of the `{e}` in the logging statement since it will get printed out anyway?